jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -106,6 +106,7 @@
       <unit id="com.fasterxml.jackson.jaxrs.jackson-jaxrs-json-provider" version="2.9.9.v20190906-1522"/>
       <unit id="org.codehaus.jackson.core" version="1.6.0.v20101005-0925"/> <!-- needed by SpringIDE -->
       <unit id="org.codehaus.jackson.mapper" version="1.6.0.v20101005-0925"/> <!-- needed by SpringIDE -->
+      <unit id="org.apache.commons.codec" version="1.6.0.v201305230611"/> <!-- needed by SpringIDE -->
```

Comment:
We just moved jbosstools-openshift with commons-codec 1.13 because of a security issue, can't we have this version ?

Reply:
Seems possible to do so, there is no strict version for codec 1.6.